.github/workflows/test.yml

```diff
@@ -6,10 +6,14 @@ on:
       - main
       - release/**
       - "*_dev"
+    paths-ignore:
```

Comment:
according to [GitHub docs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-excluding-paths), this means "run this workflow only if a file *outside* the `conformance` directory has been changed" which is the behavior we want for this workflow.

we want to run the `conformance` workflow on its own when all the changes are inside the `conformance` directory and we want to run *both* workflows when something changes outside the `conformance` directory